code/obj/flock/structure/rift.dm

```diff
@@ -56,3 +56,9 @@
 	else
 		var/severity = round(((build_time - elapsed)/build_time) * 5)
 		animate_shake(src, severity, severity)
+
+/obj/flock_structure/rift/disposing()
+	var/mob/living/intangible/flock/flockmind/F = src.flock?.flockmind
```

Comment:
no single letter variable names thanks :)

Reply:
oh man, don't look at the rest of the flock code :sheltersweat:

Reply:
me doing exactly what I tell people not to do in code review, whoops